app/javascript/components/batch_update/components/App.js

```diff
@@ -351,6 +352,29 @@ class App extends React.Component {
     return commonGenes
   }
 
+  getCommonTags() {
+    const { selectedArtworkIds, artworks } = this.state
+    window.$debug = artworks.filter(a => selectedArtworkIds.indexOf(a.id) > -1)
+
+    const tagCounts = artworks
+      .filter(artwork => selectedArtworkIds.indexOf(artwork.id) > -1)
+      .map(artwork => artwork.tags)
+      .reduce((acc, tags) => {
+        tags.forEach(t => {
+          if (!acc[t]) {
+            acc[t] = 1
+          } else {
+            acc[t] = acc[t] + 1
+          }
+        })
+        return acc
+      }, {})
```

Comment:
Neat, I like the zero dependency approach. But FYI this class already imports `lodash.intersection` which would simply this a lot.

Reply:
Oh gotcha, sorry, yeah I'll make this change!